src/main/java/com/amplitude/Event.java

```diff
@@ -9,56 +9,162 @@
 
 public class Event {
 
+  /** A unique identifier for your event. */
   public String eventType;
```

Comment:
We should have another empty line after each variable define.

Reply:
For the format part, maybe @dantetam can also run the google formatter on local to reflect? I guess that can help the formats.

Reply:
Ran the formatter. No spaces needed from them, but I decided to add the spaces for better readability.

Later on, customers will use the Javadocs to figure out which fields of `Event` to set.